components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -92,7 +92,7 @@
         [visible]="deleteModalVisible"
         objectNoun="project"
         [objectName]="projectToDelete?.id"
-        [moreDetails]="inUseMessage()"
```

Comment:
what was this for?

Reply:
That was "for future use" in this project list view (it is used, though, for policy list) to give messaging when a resource is in use; did not need it with what I implemented (but now with Susan's changes, may have to revisit that).